core/src/sql/statements/insert.rs

```diff
@@ -124,3 +139,48 @@ impl fmt::Display for InsertStatement {
 		Ok(())
 	}
 }
+
+fn iterable(id: Thing, v: Value, relation: bool) -> Result<Iterable, Error> {
+	match relation {
+		false => Ok(Iterable::Mergeable(id, v)),
+		true => {
+			let r#in = match v.pick(&*IN) {
```

Comment:
Perhaps this could be something else? `_in` or `inside` or `rin` (vs `rout`)

Reply:
Yeah, that's a fair point. Will push that up!